src/plays/emojipedia/Emojipedia.js

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState, useRef } from "react";
+import toast, { Toaster } from "react-hot-toast";
+import { FiSearch } from "react-icons/fi";
+
+// Project local imports
+import PlayHeader from "common/playlists/PlayHeader";
+import useFetch from "common/hooks/useFetch";
+import EmojiCard from "./components/EmojiCard";
+import SkeletonCard from "./components/SkeletonCard";
+import { API_BASE_URL } from "./config";
+
+function Emojipedia(props) {
+  // Your Code Start below.
+  const [emojisList, setEmojisList] = useState([]);
+  const [query, setQuery] = useState("");
+  const inputRef = useRef(null);
+
+  // Fetch all the emojis
+  const { data, loading, error } = useFetch(API_BASE_URL);
+
+  const handleEmojiSearch = (e) => setQuery(e.target.value);
+
+  // Copy emoji handler
+  const handleCopyEmoji = async (emojiCharacter) => {
+    if ("clipboard" in navigator) {
+      await navigator.clipboard.writeText(emojiCharacter);
+      toast.success(`Emoji ${emojiCharacter} copied to clipboard.`);
+    }
+  };
+
+  // Search filter
+  useEffect(() => {
+    const delayFn = setTimeout(() => {
+      if (query !== "") {

```

Comment:
Here you can directly write:
`if (!query)`

Reply:
done 👍